state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -633,18 +641,32 @@ pub mod pallet {
 			Self::handle_broadcast_failure(broadcast_id, reporter.into())?;
 			Ok(().into())
 		}
+
+		/// Re-sign and optionally re-send a broadcast request.
+		///
+		/// Requires governance origin.
+		#[pallet::call_index(5)]
+		#[pallet::weight(Weight::zero())]
+		pub fn resign_aborted_broadcast(
```

Comment:
It feels like there should a particular unit test for this

Reply:
Done.